### PR TITLE
Remove species yml file

### PR DESCRIPTION
**Pull request by vinay-ebi**
Draft PR

### CODE REVIEW

File: modules/Bio/EnsEMBL/Production/Pipeline/Webdatafile/GenomeInfoYml.pm [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+use Bio::EnsEMBL::Registry;
+use base qw/Bio::EnsEMBL::Production::Pipeline::Common::Base/;
+use YAML::XS 'LoadFile';
+use Array::Utils qw(intersect);
+
+sub write_output {
+
+  my ($self) = @_;
+  my $genomeinfo_yml = $self->param('genomeinfo_yml');
```
Comment (marcoooo) [inline modules/Bio/EnsEMBL/Production/Pipeline/Webdatafile/GenomeInfoYml.pm:41]: Why still load a .yaml file? Is it generated from elsewhere?

Comment (vinay-ebi) [reply modules/Bio/EnsEMBL/Production/Pipeline/Webdatafile/GenomeInfoYml.pm:41]: this file not used i will remove it in future commits

Comment (marcoooo) [reply modules/Bio/EnsEMBL/Production/Pipeline/Webdatafile/GenomeInfoYml.pm:41]: Fine :-)

**Comment by vinay-ebi** [comment]
will raise against master

**Pull request closed without merging**